LandSandBoat/server
# [Item Mods] Create item level pet mods (Avatar, Wyvern, Automaton)

**_I affirm:_**

## What does this pull request do?

Adds item level item mods for Avatars, Wyverns, and Automatons.
Adds supporting database entries to items with mods created above.
Adds missing mods to pet level related items.
Adds supporting core work to allow pets to exceed level 99 when spawned.

**References:**
Issue #1833 
Avatar: Lv.119 - https://www.bg-wiki.com/ffxi/Arasy_Sachet
Avatar: Lv.+2 - https://www.bg-wiki.com/ffxi/Nirvana_(Level_119)
Carbuncle: Lv.+1 - https://www.bg-wiki.com/ffxi/Asteria_Mitts_%2B1
Cait Sith: Lv.+1 - https://www.bg-wiki.com/ffxi/Lamassu_Mitts_%2B1
Wyvern: Lv.+1 - https://www.bg-wiki.com/ffxi/Dragoon%27s_Collar
Automaton: Lv.119 - https://www.bg-wiki.com/ffxi/Animator_Z
Automaton: Lv.+1 - https://www.bg-wiki.com/ffxi/Mpaca%27s_Boots

## What does this pull request not do?

Does not adjust pet stat scaling beyond level 99 (additional information/research/testing required).



## Steps to test these changes

Upload changes to your test server.
Enter the game and !additem itemID to give yourself an item with the mods above.
Change to SMN/DRG/PUP and summon an avatar/wyvern/automaton without the gear equipped.
Target the pet and !getstats to check the pet's level.
Dismiss the pet, !reset to recover timers, and equip the level affecting equipment.
Summon your pet and !getstats to check the pet's level.
